ctapipe/image/muon/tests/test_muon_ring_finder.py

```diff
@@ -33,9 +33,9 @@ def test_chaudhuri_kundu_fitter():
     yc_fit = muon_ring_parameters.ring_center_y
     r_fit = muon_ring_parameters.ring_radius
 
-    assert np.isclose(xc_fit * flashcam_focal_length / u.m / u.rad, center_xs / u.m, 1e-1)
-    assert np.isclose(yc_fit * flashcam_focal_length / u.m / u.rad, center_ys / u.m, 1e-1)
-    assert np.isclose(r_fit * flashcam_focal_length / u.m / u.rad, ring_radius / u.m, 1e-1)
+    assert u.isclose((xc_fit * focal_length).to_value(u.m*u.rad), center_xs.to_value(u.m), 1e-1)
+    assert u.isclose((yc_fit * focal_length).to_value(u.m*u.rad), center_ys.to_value(u.m), 1e-1)
+    assert u.isclose((r_fit * focal_length).to_value(u.m*u.rad), ring_radius.to_value(u.m), 1e-1)
```

Comment:
if you use `u.isclose` you do not need to strip the units! Also, comparing something as equal that has different units looks strange, what is going on here?

Reply:
It is true that it could be as simple as that when I use meter for all of the inputs and outputs. Does that look okay?

Reply:
In the end, it should probably look just like this: `u.isclose(r_fit, ring_radius, 1e-1)`

Reply:
yes exactly

Reply:
I think this can be resolved.